### DISCUSSION TITLE
Prevent parcel from resolving url in scss

### DISCUSSION THREAD

**Discussion by thgross** [parcel-bundler/parcel#6145] [Q&A] [answered]
Hi all,

Is there a way to prevent asset resolution for a specific css url value, like in this scss code:

```
#logo {
    background : url('/static/images/logo.svg');
}
```

The image only exists on the target server, and I just want parcel to write that exact url to the target css file. I can't find out how to do this. Parcel always shows this error:
`Error: ENOENT: no such file or directory, open '/srv/www/html/application/build/static/images/logo.svg'`
because the file does not exist in the build-directory.

Thanks!

**Comment by mischnic** [comment, marked as answer]
At the moment: no

A resolver plugin might be able to do this
